src/android/Library/src/MultiImageChooserActivity.java

```diff
@@ -456,21 +492,7 @@ public View getView(int pos, View convertView, ViewGroup parent) {
 
             final int id = imagecursor.getInt(image_column_index);
             final int rotate = imagecursor.getInt(image_column_orientation);
-            if (isChecked(pos)) {
-                if (android.os.Build.VERSION.SDK_INT>=16) {
-                  imageView.setImageAlpha(128);
-                } else {
-                  imageView.setAlpha(128);	
-                }
-                imageView.setBackgroundColor(selectedColor);
-            } else {
-                if (android.os.Build.VERSION.SDK_INT>=16) {
-                  imageView.setImageAlpha(255);
-                } else {
-                  imageView.setAlpha(255);	
-                }
-                imageView.setBackgroundColor(Color.TRANSPARENT);
-            }
```

Comment:
If we're trying to copy the look of Google Photos (as per the screenshot in the Jira ticket), the images there become smaller when they're checked and have a white border.

![screenshot_20170517-161856](https://user-images.githubusercontent.com/960744/27269985-f00bdef0-546f-11e7-8611-c55f2807a572.png)


You might want to check with Brian to make sure that's what we should be doing here too (I personally really like it), but if so, that would probably be done in place of these `imageView.setAlpha()` here?